src/common/components/vanilla-modules/icons/index.js

```diff
@@ -0,0 +1,202 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import styles from '../../../style/vanilla/css/icons.css';
+
+// Creates icon style class name
+const iconStyle = (icon = '') => {
+  icon = icon ? '--' + icon : '';
```

Comment:
@richmccartney I understand that, my point here is just that `icon` name is not very clear here, because it doesn't mean 'icon' but modifier to icon class.

So my only suggestion is to change `icon` to `modifier`:

```js
const iconStyle = (modifier = '') => {
  modifier = modifier ? `--${modifier}` : '';
  // ...
}
```